flycheck.el

```diff
@@ -3072,7 +3084,7 @@ nil."
                      "")))
                 (`interrupted "-")
                 (`suspicious "?"))))
-    (concat " FlyC" text)))
+    (concat " " flycheck-mode-line-lighter-base text)))
```

Comment:
I think the space is always needed because Emacs doesn't add it automatically between mode lighters. But maybe some people want to save 1 space too? I'm not sure.